scheduler/pkg/store/memory.go

```diff
@@ -311,10 +311,10 @@ func (m *MemoryStore) updateLoadedModelsImpl(
 		return nil, fmt.Errorf("failed to find server %s", serverKey)
 	}
 	for _, replica := range replicas {
-		_, ok := server.replicas[replica.GetReplicaIdx()]
+		_, ok := server.replicas[replica.replicaIdx]
```

Comment:
is there a reason why not to use the public method?

Reply:
There's mixed usage of  `replicaIdx` as a param, var and via the public method, and I was making that uniform to read.